main.go

```diff
@@ -451,7 +451,7 @@ func setup(ctx context.Context, deps *setupDeps) bool {
 	deps.vmalert.RequestConfigurationUpdate()
 
 	// Alertmanager is special due to being added to PMM with invalid /etc/alertmanager.yml.
-	deps.alertmanager.GenerateBaseConfigs(ctx)
+	deps.alertmanager.GenerateBaseConfigs()
```

Comment:
Why not just replace invalid `/etc/alermanager.yml` with valid one at PMM image creation?

Reply:
Because it should also work for various update methods

Reply:
Ah, got it!